core/src/main/scala/geotrellis/server/core/maml/CogDefinition.scala

```diff
@@ -0,0 +1,51 @@
+package geotrellis.server.core.maml
+
+import geotrellis.server.core.cog.CogUtils
+
+import com.azavea.maml.ast.{Expression, Literal, MamlKind, RasterLit}
+import com.azavea.maml.eval.tile._
+import io.circe._
+import io.circe.generic.semiauto._
+import cats.effect._
+import cats.syntax.all._
+import geotrellis.raster._
+import geotrellis.vector.Extent
+
+import java.net.URI
+
+
+case class CogNode(uri: URI, band: Int, celltype: Option[CellType])
+
+object CogNode {
+   private val intNdTile = IntConstantTile(NODATA, 256, 256)
```

Comment:
wondering is it really good to return nodata tiles? leaflet recognizes `404` err code. (`Option[Tile]` for instance)